controller/src/main/java/org/jboss/as/controller/AttributeDefinition.java

```diff
@@ -81,7 +81,7 @@
     private final ParameterCorrector valueCorrector;
     private final ParameterValidator validator;
     private final EnumSet<AttributeAccess.Flag> flags;
-    protected final AttributeMarshaller attributeMarshaller;
+    protected final AttributeMarshaller marshaller;
```

Comment:
good point, will revert this, but will also look into removing bunch of deprecated constructors in AttributeDefinition and its inheritance chain, as we moved to builder api long time ago and I think they should be ripe to be removed.